website/other/bezier-rs-demos/wasm/src/bezier.rs

```diff
@@ -365,7 +365,7 @@ impl WasmBezier {
 	}
 
 	pub fn rotate(&self, angle: f64) -> WasmBezier {
-		WasmBezier(self.0.rotate(angle))
+		WasmBezier(self.0.rotate_about_point(angle, DVec2::new(100., 100.)))
```

Comment:
It doesn't make a significant difference because these functions aren't user-facing, but I don't like the second argument being hard-coded here. I think we should be defining the point around which to rotate in `App.vue`